### PR TITLE
Added README.md and Makefile.

### PR DESCRIPTION
**Pull request by ypid**
You might want to put the project under a license.

### CODE REVIEW

**Comment by patrickod** [comment]
Added LICENSE in #13

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+stop:
+	docker stop tor
+
+run: tor
+
+tor:
+	-@docker rm -f "$@"
+	docker run -d \
+		--name "$@" \
+		-e "TZ=Europe/Berlin" \
```
Comment (patrickod) [inline Makefile]: I'm guessing this is your locale at the moment but I think we should take this out before merging for folks?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		-p 993:993 \
+		-p 465:465 \
```
Comment (patrickod) [inline Makefile]: I'm guessing that you allow exit traffic from these ports?

File: Makefile (hunk 1 above)
Comment (ypid) [inline Makefile]: I hope that users which use the Makefile will look in it first and read the comments which I just added. If you would like to take it out, please do so.

**Comment by ypid** [comment]
Ready for merging?

**Pull request merged**